README.md

```diff
@@ -1,11 +1,109 @@
 # masq-lib
+
+## Install
+```
+npm install masq-lib --save
+```
+
+## Launch Test
 ```
 npm install
 npm test
 ```
 
-Analyze bundle size with:
+## Analyze bundle size with:
 ```
 npm run stats
 npm run analyze
 ```
+
+
+## Usage
+
+``` js
+const Masq = require('masq-lib')
+
+const masq = new Masq(
+    'TestApp',
+    'Some User-app',
+    'http://www.some-user-app.com/logo.png'
+)
+
+if (masq.isLoggedIn()) {
+    await masq.connectToMasq()
+} else {
+    const stayConnected = false
+    const { link } = await masq.logIntoMasq(stayConnected)
+    // Display link to the user
+    await masq.logIntoMasqDone()
+
+    await masq.put('someKey', 'someValue')
+    const val = await masq.get('someKey')
+}
+```
+
+## API
+
+#### `const masq = new Masq(appName, appDescription, appLogoURL)`
+
+Create a new instance of Masq.
+
+`appName` is the name of the app.
+
+`appDescription` is a short description of the app.
+
+`appLogoURL` is a link to a logo of the app.
+
+
+#### `masq.isLoggedIn() : boolean`
+
+Returns a boolean, true if a User is already logged into Masq, false if not.
+
+#### `masq.isConnected() -> connected: boolean`
+
+Returns a boolean, true if a User is already logged into Masq and if the User-app is already connected to Masq.
+
+#### `masq.connectToMasq() -> Promise<void>`
+
+Connects to Masq if a User is already logged into Masq.
```

Comment:
Can you add just a sentence which describe what 'logged into Masq' mean please ? Like: `(e.g: isLoggedIn() returns true)`